### DISCUSSION TITLE
Welcome to FED3 Discussions!

### DISCUSSION THREAD

**Discussion by KravitzLab** [KravitzLabDevices/FED3#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jaya2075** [comment]
Great work on FED3 device. Can this device be used for  monitoring rat food intake? And may I know the dimension of this  device please? Thank you :)

**Comment by KravitzLab** [comment]
Thanks!  There are some groups using FED3 with juvenile rats, but it was really designed for mice so I don't know if it will work for adult rats.

Dimensions are other specs are here: https://github.com/KravitzLabDevices/FED3/wiki/FED3-Specifications

Best,
-Lex

________________________________

**Reply by Jaya2075** [reply]
Thanks Lex. Is the product available for purchasing if required?

**Comment by KravitzLab** [comment]
Yup it's being sold by Open Ephys:
https://open-ephys.org/fed3/fed3
[http://static1.squarespace.com/static/53039db8e4b0649958e13c7b/5da3b78ee91096688f5cdf9e/5da3b7f835c26c463cc2427d/1652721057685/fed3.1_4.jpg?format=1500w]<https://open-ephys.org/fed3/fed3>
FED3.1 Wire-free pellet-dispensing — Open Ephys<https://open-ephys.org/fed3/fed3>
FED3.1 is a pellet-dispensing device for training mice on operant tasks. For an introductory video, click here. To maximize flexibility, FED3.1 is open-source, compact, and wireless, so that it can be placed in the home cage.
open-ephys.org
I'm not financially involved with their business but they're friends of mine.  They can answer any questions about purchasing it!  Best, -Lex
________________________________

**Reply by Jaya2075** [reply]
Excellent. Thank you Lex, Much appreciated. Would I be able to get some contact details of the seller please? Or I am happy to share my email address (jaya.jayaseelan1@monash.edu . Thanks again for your time.

**Comment by KravitzLab** [comment]
Sure! You can reach out to:
***@***.***

Best, -Lex

________________________________

**Reply by Jaya2075** [reply]
Hi Lex, Are there any information about heat generated by the FED 3 device in a closed cage (microisolator) environment over the 7 days usage? Thanks, Jaya

**Comment by KravitzLab** [comment]
I don't think it generates any heat, although haven't tested this explicitly.  But we do have some temp records in cages with FED3 and they seem normal.  Here's some temp recordings we did.  Does this answer your question?

[cid:e5cd702f-bce2-425f-b1db-b503b645a904]
________________________________

**Reply by Jaya2075** [reply]
Thaks Lex.  Sorry, I couldn't see any temperature recordings.  Is there a link or file to view the content?

**Comment by KravitzLab** [comment]
Here you go, this was from 5 cages with FEDs running in them.  BTW I'm not sure the 1C difference between the cages is real, our temp sensors have a 1C tolerance.

![Untitled](https://user-images.githubusercontent.com/16770585/217146291-b8f8317f-4b18-40a3-869a-890ce816c43e.png)

**Reply by Jaya2075** [reply]
Thank you Lex. Thats very useful and much appreciated. One more question. Do you know if the cages used for the above experiment were a fully closed microisolator cages or normal mouse cages with the cage top? Best Regards

**Comment by KravitzLab** [comment]
Yup these were Allentown NextGen cages with closed tops, housed on a ventilated rack

**Reply by Jaya2075** [reply]
Excellent. Thank you very much

**Comment by Jaya2075** [comment]
Hi Lex, I hope you are doing well. Just wondering what would be the largest food pellet size that can be used it in FED3 device.  I recall for mouse it's a 20 mg food pellet. But if the FED3 is used for juvenile rats, can we increase the size of the food pellet? Also, do you know from where I can purchase the food pellets? Thanks

**Comment by KravitzLab** [comment]
Hi, FED3 was designed for 20mg pellets so it likely won't work with larger pellets.  You can find information on pellets here: https://github.com/KravitzLabDevices/FED3/wiki/Pellet-information

You also may want to ask this question on the forum, maybe someone else has tried it!
https://groups.google.com/g/fedforum

________________________________

**Reply by Jaya2075** [reply]
Thanks Lex.
